polytope_server/common/request_store/dynamodb_request_store.py

```diff
@@ -181,6 +187,55 @@ def remove_request(self, id):
 
         logger.info("Request ID %s removed.", id)
 
+    def revoke_request(self, user: User, id: str):
+        if id == "all":
+            # Revoke all requests of the user that are waiting or queued
+            response = self.table.scan(
```

Comment:
You can query the status-index to improve the performance and reduce the billed amount per transaction